tests/regression/cfg/util/cfgDot.ml

```diff
@@ -0,0 +1,73 @@
+(* Part of CilCfg *)
+class allBBVisitor = object (* puts every instruction into its own basic block *)
+  inherit GoblintCil.nopCilVisitor
+  method! vstmt s =
+    match s.skind with
+    | Instr(il) ->
+      let list_of_stmts =
+        List.map (fun one_inst -> GoblintCil.mkStmtOneInstr one_inst) il in
+      let block = GoblintCil.mkBlock list_of_stmts in
+      ChangeDoChildrenPost(s, (fun _ -> s.skind <- Block(block); s))
+    | _ -> DoChildren
+
+  method! vvdec _ = SkipChildren
+  method! vexpr _ = SkipChildren
+  method! vlval _ = SkipChildren
+  method! vtype _ = SkipChildren
+end
```

Comment:
Any particular reason this cannot be reused from `CilCfg`?

Such duplication is a bit bad because it may cause the test program and our actual program to go out of sync, which would mean tests still passing even if things are broken.

Reply:
`CilCfg` has some things that depend on `Autotune` which is (and has to be) in `goblint.lib`. I'll see if I can do anything about it.

Reply:
I extracted `CilCfg0` for now.